crates/trie/sparse/Cargo.toml

```diff
@@ -38,6 +38,7 @@ reth-testing-utils.workspace = true
 reth-trie = { workspace = true, features = ["test-utils"] }
 reth-trie-common = { workspace = true, features = ["test-utils", "arbitrary"] }
 reth-trie-db = { workspace = true, features = ["test-utils"] }
+reth-tracing.workspace = true
```

Comment:
reth-tracing is not a dev dependency so that it works for tests. Otherwise, we use the no_std version of tracing